geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/coprocessor/KryoLazyDensityCoprocessor.scala

```diff
@@ -122,52 +128,44 @@ class KryoLazyDensityCoprocessor extends KryoLazyDensityService with Coprocessor
 
 object KryoLazyDensityCoprocessor extends KryoLazyDensityUtils {
 
-  private val SFT_OPT = "sft"
+  private val SFT_OPT    = "sft"
   private val FILTER_OPT = "filter"
   private val RANGES_OPT = "ranges"
+  private val SCAN_OPT = "scan"
 
   /**
     * Creates an iterator config for the kryo density iterator
     */
   def configure(sft: SimpleFeatureType,
-                ranges: Seq[Scan],
-                filters: Seq[HFilter],
+                scan: Scan,
+                filterList: FilterList,
                 hints: Hints): Map[String, String] = {
     import org.locationtech.geomesa.index.conf.QueryHints.RichHints
     val envelope = hints.getDensityEnvelope.get
     val (width, height) = hints.getDensityBounds.get
     val weight = hints.getDensityWeight
-    configure(sft, ranges, filters, envelope, width, height, weight)
+    configure(sft, scan, filterList, envelope, width, height, weight)
   }
 
 
   protected def configure(sft: SimpleFeatureType,
-                          ranges: Seq[Scan],
-                          filters: Seq[HFilter],
+                          scan: Scan,
+                          filterList: FilterList,
                           envelope: Envelope,
                           gridWidth: Int,
                           gridHeight: Int,
                           weightAttribute: Option[String]): Map[String, String] = {
     val is = mutable.Map.empty[String, String]
-    val rangeBuilder : StringBuilder = StringBuilder.newBuilder
 
     is.put(ENVELOPE_OPT, s"${envelope.getMinX},${envelope.getMaxX},${envelope.getMinY},${envelope.getMaxY}")
     is.put(GRID_OPT, s"$gridWidth,$gridHeight")
     weightAttribute.foreach(is.put(WEIGHT_OPT, _))
     is.put(SFT_OPT, SimpleFeatureTypes.encodeType(sft, includeUserData = true))
-
```

Comment:
Yes, it was my intent to configure the Density process with a Scan and FilterList.